review_process.md

```diff
@@ -1,68 +1,65 @@
-the application.
+Granting Entities will then perform an initial review using the
+following [evaluation criteria](./evaluation_criteria.md),
+and solicit additional peer review where appropriate.
 
-**NOTE:** At this point, no funding has yet been approved. Some
-back and forth may occur between the first call and next step.
+Each proposal will then be matched with a *Steward*, who will serve
+as the Applicant's primary point of contact.
 
-## Approval by the Board of Management (BoM)
+Once assigned, Stewards will setup an initial call with the Applicant
+to clarify outstanding questions about their submission and revise
+the proposed scope, timeline, and budget as needed. Together they 
+will draft a prospective *Financing Agreement*, which will used as 
+the basis for subsequent evaluation.
 
-At some point in the previous step, the BoM will have given unanimous
-approval to fund certain projects, for which an Executive Summary is
-created. All the projects approved by the BoM will then be assembled
-in a package sent to the Foundation Council for review. This Executive
-Summary can be found at the bottom of the [Evaluation Criteria](./evaluation_criteria.md).
+## Board of Management & Technical Advisory Committee
 
-## Recommendation to the Foundation Council
+The draft Financing Agreement, together with Granting Entity reviews
+and the original application are delivered to the *Board of
+Management*, a supervisory body employed by the Interchain Foundation.
 
-The Foundation Council reviews each project and either approves or
-denies the funding. Given the due diligence that has occurred thus far,
-the FC must give good reasons for denying funding.
+The Board of Management will administer the assembly of a *Technical
+Advisory Committee*, composed of domain experts with a record
+of community contribution. This committee will provide feedback
+on each proposal's technical soundness and benefit to the wider
+ecosystem.
 
-## Approval by the Foundation Council
+The Board of Management will then incorporate feedback from the 
+Technical Advisory Committee with all previous materials, to create
+a final package which is either unanimously endorsed or returned 
+to Stewards with revising comments.
 
-Once the Foundation Council approves the funding, agreements are drafted and finally signed.
+## Foundation Council
 
-## Timeline
+Endorsed packages are then sent to the *Foundation Council*,
+who will certify their final approval by signing the 
+Financing Agreement, or exercising a final veto. Given the 
+extent of prior due diligence, the Foundation Council must 
+provide a strong justification to deny final approval.
 
-This process typically takes 3-8 weeks. 
+## Common Reasons a Proposal is _Not_ Funded
 
-## Common Not Funded Reasons
+- Incomplete Scope, Budget, or Timeline
+- Asking amount is too high
+- Timing (too early for Interchain Foundation consideration)
+- Previously supported similar work
```

Comment:
maybe something like "redundant efforts" since it could be deemed to be unnecessary in light of another project even if that other project wasn't funded by ICF?

Reply:
changed